src/applications/mhv-secure-messaging/containers/Compose.jsx

```diff
@@ -89,6 +104,53 @@ const Compose = () => {
     },
     [header, acknowledged, pageTitle],
   );
+  // make sure the thread list is fetched when navigating to the compose page
+  useEffect(
+    () => {
+      const loadSentFolder = () => {
+        dispatch(
+          getListOfThreads(
+            DefaultFolders.SENT.id,
+            100,
+            1,
+            threadSortingOptions.SENT_DATE_DESCENDING.value,
+            false,
+          ),
+        );
+      };
+
+      if (!isLoading && !threadList) {
```

Comment:
context: this had to be split because, even with AI's help, the logic here was always clashing and being wrong. so for the sake of readability and shipping, I broke it out into two separate statements

Reply:
I think this is potentially running into infinite loop if `getListOfThreads` does not resolve a valid response